cmd/ceremony/main.go

```diff
@@ -615,10 +650,15 @@ func rootCeremony(configBytes []byte) error {
 	if !bytes.Equal(lintCert.RawSubject, lintCert.RawIssuer) {
 		return fmt.Errorf("mismatch between self-signed lintCert RawSubject and RawIssuer DER bytes: \"%x\" != \"%x\"", lintCert.RawSubject, lintCert.RawIssuer)
 	}
-	_, err = signAndWriteCert(template, template, lintCert, keyInfo.key, signer, config.Outputs.CertificatePath)
+	finalCert, err := signAndWriteCert(template, template, lintCert, keyInfo.key, signer, config.Outputs.CertificatePath)
+	if err != nil {
+		return err
+	}
+	err = postIssuanceLinting(finalCert, config.SkipLints)
 	if err != nil {
 		return err
 	}
+	log.Printf("Post issuance linting completed for %s\n", config.Outputs.CertificatePath)
```

Comment:
The log output in these ceremony functions is weirdly inconsistent. Why are opening the hsm slot and completing linting worthy of log lines, but generating the cert itself isn't?

Anyway I don't know if it's wrong to add a log line here but it does feel a bit weird given all the things we don't log.

Reply:
@aarongable. For purposes of post ceremony inspection and documentation that zlint was run against the generated certificate with no errors I asked Phil to add something like this. Honestly, I don't mind if it is this or an output file containing the zlint output. Actually, the more I type this the more I think I would rather request an output file of the successful zlint along side each certificate.

Reply:
Makes sense to have positive confirmation. I don't think an output file would do much; zlint doesn't provide a summary of "we ran all these lints and they all passed", it just provides a (hopefully empty) error object showing all the ones which didn't.